plugin/code_actions.py

```diff
@@ -79,7 +79,7 @@ def request_with_diagnostics_async(
         region: sublime.Region,
         session_buffer_diagnostics: Sequence[Tuple[SessionBufferProtocol, Sequence[Diagnostic]]],
         actions_handler: Callable[[CodeActionsByConfigName], None]
-    ) -> CodeActionsCollector:
+    ) -> Optional[CodeActionsCollector]:
```

Comment:
Since we never use the return value anyway, might as well return None?

Reply:
True